drivers/pds/lib/utils.go

```diff
@@ -1024,6 +1026,129 @@ func RunTpccWorkload(dbUser string, pdsPassword string, dnsEndpoint string, dbNa
 	return flag
 }
 
+// Creates a temporary non PDS namespace of 6 letters length randomly chosen
+func CreateTempNS() (string, bool) {
+	length := 6
+	rand.Seed(time.Now().UnixNano())
+	b := make([]byte, length)
+	rand.Read(b)
+	namespace := fmt.Sprintf("%x", b)[:length]
+	ns := &corev1.Namespace{
+		ObjectMeta: metav1.ObjectMeta{
+			Name: namespace,
+		},
+	}
+	ns, err = k8sCore.CreateNamespace(ns)
+	if err != nil {
+		log.Errorf("Error while creating namespace %v", err)
+		return "", false
+	}
+	return namespace, true
+}
+
+// Create a Persistent Vol of 5G manual Storage Class
+func CreateIndependentPV(name string) bool {
+	pv := &corev1.PersistentVolume{
+
+		TypeMeta: metav1.TypeMeta{Kind: "PersistentVolume"},
+		ObjectMeta: metav1.ObjectMeta{
+			Name: name,
+		},
+
+		Spec: corev1.PersistentVolumeSpec{
+			StorageClassName: "manual",
+			AccessModes: []corev1.PersistentVolumeAccessMode{
+				"ReadWriteOnce",
+			},
+			Capacity: corev1.ResourceList{
+				corev1.ResourceName(corev1.ResourceStorage): resource.MustParse("5Gi"),
+			},
+			PersistentVolumeSource: corev1.PersistentVolumeSource{
+				HostPath: &corev1.HostPathVolumeSource{
+					Path: "/mnt/data",
+				},
+			},
+		},
+	}
+	_, err := k8sCore.CreatePersistentVolume(pv)
+	if err != nil {
+		log.Errorf("PV Could not be created. Exiting")
+		return false
+	}
+	return true
+}
+
+// Create a PV Claim of 5G Storage
+func CreateIndependentPVC(namespace string, name string) bool {
+	ns := namespace
+	storageClass := "manual"
+	createOpts := &corev1.PersistentVolumeClaim{
+		ObjectMeta: metav1.ObjectMeta{
+			Name:      name,
+			Namespace: ns,
+		},
+		Spec: corev1.PersistentVolumeClaimSpec{
+			AccessModes:      []corev1.PersistentVolumeAccessMode{corev1.ReadWriteOnce},
+			StorageClassName: &storageClass,
+			Resources: corev1.ResourceRequirements{
+				Requests: corev1.ResourceList{
+					corev1.ResourceStorage: resource.MustParse("5Gi"),
+				},
+			},
+		},
+	}
+	_, err := k8sCore.CreatePersistentVolumeClaim(createOpts)
+	if err != nil {
+		log.Errorf("PVC Could not be created. Exiting. %v", err)
+		return false
+	}
+	return true
+}
+
+// Create an Independant MySQL non PDS App running in a namespace
+func CreateIndependentMySqlApp(ns string, appImage string, pvcName string) (string, bool) {
+	namespace := ns
+	podName := "mysql-app"
+	podSpec := &corev1.Pod{
+		TypeMeta: metav1.TypeMeta{
+			Kind:       "Pod",
+			APIVersion: "v1",
+		},
+		ObjectMeta: metav1.ObjectMeta{
+			GenerateName: podName + "-",
+			Namespace:    namespace,
+		},
+		Spec: corev1.PodSpec{
+			Containers: []corev1.Container{
+				{
+					Name:  podName,
+					Image: appImage,
+					Env:   make([]corev1.EnvVar, 1),
+				},
+			},
+			RestartPolicy: corev1.RestartPolicyOnFailure,
+		},
+	}
+	volumename := "app-persistent-storage"
```

Comment:
is the pv which we created above? if so pass it as a param.

Reply:
Nopes - this would be a volume name - generic for any app